systems/controllers/iiwa_position_controller.cc

```diff
@@ -0,0 +1,86 @@
+#define NUM_JOINTS 7
+#define K_P 50
+#define K_OMEGA 50
+#define WORLDFRAME_ID 0
+#define ENDEFFECTOR_FRAME_ID 10
+
+#include "systems/controllers/iiwa_position_controller.h"
+
+namespace dairlib{
+namespace systems{
+
+KukaIiwaPositionController::KukaIiwaPositionController()
+{
+
+	// Set up this block's input and output ports
+  	// Input port values will be accessed via EvalVectorInput() later
+	joint_position_measured_port = this->DeclareVectorInputPort("joint_position_measured", BasicVector<double>(NUM_JOINTS)).get_index();
+	endpoint_position_commanded_port = this->DeclareVectorInputPort("endpoint_position_commanded", BasicVector<double>(3)).get_index();
+
+	this->DeclareVectorOutputPort(BasicVector<double>(3), &KukaIiwaPositionController::CalcOutputVelocity);
```

Comment:
These two ports should probably be combined into one port of size 6, for the **twist**, unless you see a reason why they would be connected to different destinations.